index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7a8530007f.js" crossorigin="anonymous"></script>
+    <title>Microverse</title>
+</head>
+<body>
+    <header class="intro-section">
+        
+        <div class="bar">
+            <div class="clock">9:41</div>
+            <div class="icons">
+                <i class="fa-solid fa-signal"></i>
+                <i class="fa-solid fa-wifi"></i>
+                <i class="fa-solid fa-battery-full"></i>
+            </div>
+        </div>
+    
+        <nav class="logo">
+            <div class="mylogo">My Logo</div>
```

Comment:
- [ ] I suggest that you consider having `a` tag for the logo. I you notice most websites have the logo, which when clicked redirects to the home page of the application. In addition, this is for three reasons: convention, semantics, and user experience(UX) ☺️.